packages/terra-dropdown-button/src/_DropdownList.jsx

```diff
@@ -69,6 +78,7 @@ class DropdownList extends React.Component {
       event.preventDefault();
     } else if (keyCode === KeyCode.KEY_DOWN) {
       if (!this.pressed) {
+        this.expanded = '';
```

Comment:
Can we add a unit test to see if this gets set?

Reply:
Implemented.